src/Dhall/Tutorial.hs

```diff
@@ -1551,6 +1551,24 @@ import Dhall
 -- behavior-preserving.  This provides an easy way to detect refactoring errors
 -- that you might accidentally introduce.  The hash not only protects you
 -- from attackers, but also protects against human error, too!
+--
+-- If you have a file which either doesn't already use hashed imports,
+-- or you changed some of the imports and want to update the hashes you can use the
+-- freeze command to either add or updated hashes:
+--
+-- > cat foo.dhall
+-- ''
+--    let replicate = ./r.dhall
```

Comment:
Maybe use the GitHub URL for the import so that users can reproduce the example and also so that I can later add the example to the test suite